### PR TITLE
[BUF-2022] Add Registration

### PR DESCRIPTION
**Pull request by onelioubov**
- Add registration for 2022
- Remove link to CFP as that is now over

### CODE REVIEW

File: data/events/2022-buffalo.yml [hunk 1]
```diff
@@ -20,11 +20,11 @@ cfp_date_announce: 2022-05-03 # inform proposers of status
 
 cfp_link: "https://sessionize.com/devops-days-buffalo/" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
 
-registration_date_start: # start accepting registration. Leave blank if registration is not open yet
-registration_date_end: # close registration. Leave blank if registration is not open yet.
+registration_date_start: 2022-06-13 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: 2022-06-13 # close registration. Leave blank if registration is not open yet.
```
Comment (glasnt) [inline data/events/2022-buffalo.yml:24]: You may want to change this date to when you want to stop accepting registrations.

File: data/events/2022-buffalo.yml [hunk 2]
```diff
@@ -36,10 +36,10 @@ coordinates: "42.8797755,-78.8763309" # The coordinates of your city. Get Latitu
 location_address: "1 Seneca St, Buffalo, NY 14203, USA" #Optional - use the street address of your venue. This will show up on the welcome page if set.
 
 nav_elements: # List of pages you want to show up in the navigation of your page.
-  - name: propose
+#  - name: propose
   - name: location
-#  - name: registration
-#  - url: "https://ti.to/devops-days-buffalo/devops-days-buffalo-2021"
+  - name: registration
+  - url: "https://ti.to/devops-days-buffalo/2022"
```
Comment (glasnt) [inline data/events/2022-buffalo.yml:42]: This link is currently showing "This event is not live. Check back soon!"

Comment (onelioubov) [reply data/events/2022-buffalo.yml:42]: This is intentional. We're going to make the tickets go live once the PR is merged 🙂

File: data/events/2022-buffalo.yml (hunk 1 above)
Comment (onelioubov) [reply data/events/2022-buffalo.yml]: Thanks for catching that!

File: data/events/2022-buffalo.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ cfp_date_announce: 2022-05-03 # inform proposers of status
 cfp_link: "https://sessionize.com/devops-days-buffalo/" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
 
 registration_date_start: 2022-06-13 # start accepting registration. Leave blank if registration is not open yet
-registration_date_end: 2022-06-13 # close registration. Leave blank if registration is not open yet.
+registration_date_end: 2022-07-13 # close registration. Leave blank if registration is not open yet.
```
Comment (glasnt) [inline data/events/2022-buffalo.yml:24]: Your event is scheduled for September, do you intend to close registrations in July?

Comment (onelioubov) [reply data/events/2022-buffalo.yml]: I appreciate you checking! In my haste to get the PR in, I wanted to add some arbitrary date in the future that I can go back and change later after hashing out the details. I changed it to a more appropriate one for now.

**Pull request merged**